sys/parsoid.js

```diff
@@ -735,6 +735,7 @@ class ParsoidService {
             headers: {
                 'content-type': 'application/json',
                 'user-agent': req['user-agent'],
+                'content-language': req.headers['content-language']
```

Comment:
Did you mean `accept-language` ?

Reply:
Nope, I meant `content-language` - parsoid requires that header to be set for the conversion request.